rotkehlchen/api/rest.py

```diff
@@ -3955,6 +3957,8 @@ def delete_manual_latest_price(
 
         for pair in pairs_to_invalidate:
             Inquirer().remove_cached_current_price_entry(cache_key=pair)
+        # also invalidate the usd price equivalent of the `from_asset`
+        Inquirer().remove_cached_current_price_entry(cache_key=(asset, A_USD))
```

Comment:
Isn't this already in `pairs_to_invalidate`? If I understood correctly that list contains all the pairs that have asset in any column

Reply:
> Isn't this already in pairs_to_invalidate? 

No, it isn't.  Unless a USD manual price for an asset is added to the db, it will not be on the list.

Reply:
> Isn't this already in pairs_to_invalidate? 

No, it isn't.  Unless a USD manual price for an asset is added to the db, it will not be on the list.

Reply:
ah okey. got it